docs/client.rst

```diff
@@ -289,6 +290,20 @@ Statistics
    :members:
    :special-members: __str__
 
+.. _`Logging`:
+
+Logging
+-------
+
+.. automodule:: pywbem._logging
+
```

Comment:
Add `:members:` below the automodule statement, to make the global constants show up.

When I did that, I got the documentation of the `PywbemLoggers` class twice. So maybe the autoclass statement for that class needs to be removed? You need to try that out.